tests/session/session-getTransactionState-001.phpt

```diff
@@ -0,0 +1,67 @@
+--TEST--
+MongoDB\Driver\Session::getTransactionState()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto() ?>
+<?php skip_if_no_transactions(); ?>
+<?php skip_if_not_clean(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+
+/* Create collections as that can't be (automatically) done in a transaction */
+$cmd = new \MongoDB\Driver\Command([
+    'create' => COLLECTION_NAME,
+]);
+$manager->executeCommand(DATABASE_NAME, $cmd);
+
+/* Start a session */
+$session = $manager->startSession();
+
+/* Empty transaction, and aborted empty transaction */
+var_dump($session->getTransactionState());
+$session->startTransaction();
+var_dump($session->getTransactionState());
+$session->abortTransaction();
+var_dump($session->getTransactionState());
+
+/* Empty transaction, and committed empty transaction */
+$session->startTransaction();
+var_dump($session->getTransactionState());
+$session->commitTransaction();
+var_dump($session->getTransactionState());
+
+/* Aborted transaction with one operation */
+$session->startTransaction();
+$bw = new \MongoDB\Driver\BulkWrite();
```

Comment:
Does it make sense to `var_dump()` here as well, so we get "starting" in the test output? That makes it easier to read the expected output, as would extra line breaks between the various test cases here.

An alternative is to replace the comments with `echo "\nTest case:\n";` statements, which we've done before in some tests that produce a bunch of output that wouldn't make much sense on its own.